scripts/apps/search/directives/SearchResults.ts

```diff
@@ -424,8 +424,21 @@ export function SearchResults(
 
                 if (item) {
                     if (item._type === 'externalsource') {
-                        processPreview(item);
-                        return;
+                        var item_id = item.extra ? item.extra.itemid : false;
+
+                        if (item_id && item.guid) {
+                            criteria.params = {
+                                'detailed_info': {
+                                    'newsItemId': item.extra.itemid,
```

Comment:
why do we need this? is guid not enough? this is again client specific (actually belga archive specific)

Reply:
IMO we need newsItemId to get a detailed response, so only guid is not enough.  However, we can rename key **newsItemId** to **item_id** like - 
```
'detailed_info': {
       'item_id': item.extra.itemid,
       'guid': item.guid,
}
```

Reply:
what is the difference between those 2 ids?

Reply:
guid is `newsObjectId`, which is used to call the normal API which fetches only the main information and
item_id is `newsItemId` which is used to call the detailed API which fetches proper details like the item contains related articles or not, proper authors' details, etc for Belga Archive.

Reply:
I see, could we use one to fetch the other? or maybe if we send that one as `preview_id` or something and based on that we would fetch the preview or not, otherwise this still looks belga specific in the client code